modules/social_features/social_event/config/install/core.entity_view_display.node.event.small_teaser.yml

```diff
@@ -13,50 +13,42 @@ dependencies:
     - field.field.node.event.field_event_image
     - field.field.node.event.field_event_location
     - field.field.node.event.field_files
+    - image.style.social_medium
     - node.type.event
   module:
     - datetime
+    - image
     - user
-_core:
```

Comment:
@OH17L I don't think we need to remove `default_config_hash` with a new config management flow.

Reply:
@Kingdutch I see that you did it [here](https://github.com/goalgorilla/open_social/pull/1714/commits/e249005fe9a2e3432cde0ef8cfa3e588d61d8ef8#diff-bfe5a2c29641f5ddd592c5706b6cb912L20). Can you explain why?

Reply:
I recently learned that we should remove the `default_config_hash` key as it will be calculated by the config installer when the config is imported. So it shouldn't be in the .yml files.

Previously I thought we should add it, but I was wrong.